doc/classes/Vector2.xml

```diff
@@ -325,6 +325,15 @@
 				Returns the result of reflecting the vector from a line defined by the given direction vector [param n].
 			</description>
 		</method>
+		<method name="rotate_toward" qualifiers="const">
+			<return type="Vector2" />
+			<param index="0" name="to" type="Vector2" />
+			<param index="1" name="delta" type="float" />
+			<description>
+				Returns the result of rotating this vector towards [param to], by increment [param delta] (in radians).
```

Comment:
```suggestion
				Returns the result of rotating this vector toward [param to], by increment [param delta] (in radians).
```
"towards" is [generally considered](https://en.wiktionary.org/wiki/toward#Usage_notes) UK English, and "toward" is used in the line below so best to keep uniform